numba_dpex/tests/debugging/common.py

```diff
@@ -13,7 +13,7 @@
 
 def script_path(script):
     package_path = pathlib.Path(numba_dpex.__file__).parent
-    return str(package_path / "examples/debug" / script)
+    return str(package_path / "../examples/debug" / script)
```

Comment:
The change will not work as is. Once you move the examples out of tree, any one who installs the package will not be able to run tests from the package as the example path will be invalid.

I suggest moving whatever source files that the debug tests use from the `examples/debug` into the `tests/debugging` directory.

Reply:
Moved files to tests/debugging and updated path.